incubator/spark-history-server/charts/nfs/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: nfs
```

Comment:
Would we be able to use the [stable NFS server](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner) instead of this one? I ask without knowing the details on that one.

Reply:
Hmm that's a good suggestion. But the current configurations of the chart are intended to provide the user with a smooth and working-out-of-box experience. As documented in `README` of the chart, there are some restrictions to using the PVC approach. I would expect that most users would want to go for HDFS or GCS option in production with this chart.

Therefore as a demo purpose, I think the current approach suffices. The sub-chart only installs an NFS server. The stable NFS server chart also configures a dynamic provisioner, which I think is a nice feature but doesn't necessarily help the user get started using this chart.